config/crd/bases/machinelearning.seldon.io_seldondeployments.yaml

```diff
@@ -0,0 +1,7977 @@
+
```

Comment:
Is there a way that ask kubebuilder does not generate the seldon CRD? or this will be copied to primehub chart by the makefile.

Reply:
delete it